xbmc/addons/addoninfo/AddonInfoBuilder.cpp

```diff
@@ -117,6 +117,8 @@ bool CAddonInfoBuilder::ParseXML(const AddonInfoPtr& addon, const TiXmlElement*
     return false;
   }
 
+  const bool isRepoXMLContent = !repo.artdir.empty() && !repo.datadir.empty();
```

Comment:
This should have a comment explaining why this means it’s repo xml content.

Reply:
Added comment